src/main/kotlin/ar/edu/unahur/obj2/vendedores/Vendedor.kt

```diff
@@ -28,24 +31,39 @@ abstract class Vendedor {
   fun otrasCertificaciones() = certificaciones.count { !it.esDeProducto }
 
   fun puntajeCertificaciones() = certificaciones.sumBy { c -> c.puntaje }
+  fun esGenerico() = certificaciones.any{ !it.esDeProducto }
 }
 
 // En los parámetros, es obligatorio poner el tipo
 class VendedorFijo(val ciudadOrigen: Ciudad) : Vendedor() {
   override fun puedeTrabajarEn(ciudad: Ciudad): Boolean {
     return ciudad == ciudadOrigen
   }
+  override fun esInfluyente() = false
 }
 
 // A este tipo de List no se le pueden agregar elementos una vez definida
 class Viajante(val provinciasHabilitadas: List<Provincia>) : Vendedor() {
   override fun puedeTrabajarEn(ciudad: Ciudad): Boolean {
     return provinciasHabilitadas.contains(ciudad.provincia)
   }
+
+  override fun esInfluyente(): Boolean {
+    return provinciasHabilitadas.sumBy { it.poblacion } >= 10000000
+  }
 }
 
 class ComercioCorresponsal(val ciudades: List<Ciudad>) : Vendedor() {
   override fun puedeTrabajarEn(ciudad: Ciudad): Boolean {
     return ciudades.contains(ciudad)
   }
+  fun provincias(): List<Provincia> {
+    return ciudades.map { it.provincia }
+  }
+  fun cantidadCiudadesConSucursales() = ciudades.toSet().size
+  fun cantidadProvinciasConSucursales() = provincias().toSet().size
+
+  override fun esInfluyente(): Boolean {
+    return this.cantidadCiudadesConSucursales() >= 5 || this.cantidadProvinciasConSucursales() >= 3
+  }
```

Comment:
2 cosas:
* Bien delegar el cálculo a otros métodos. 
* En la materia sugerimos la opción de escribir las consultas (método que devuelve) con `=` y así te ahorrás declarar el tipo de dato :
```suggestion
  override fun esInfluyente() = this.cantidadCiudadesConSucursales() >= 5 || this.cantidadProvinciasConSucursales() >= 3
```